Assets/Tests/PlayModeTests/TestUtility.cs

```diff
@@ -1,114 +1,142 @@
-﻿using Newtonsoft.Json.Linq;
-using System.Collections.Generic;
-using Plugins.CountlySDK;
-using Plugins.CountlySDK.Models;
-using Plugins.CountlySDK.Persistance.Entities;
-using Plugins.CountlySDK.Enums;
-using System.Text.RegularExpressions;
-
-namespace Assets.Tests.PlayModeTests
-{
+﻿using Newtonsoft.Json.Linq;
```

Comment:
what changed here so much? line endings?

Reply:
new imports added and it changed lines I guess, in case could you check for line endings